lms/static/sass/views/_shoppingcart.scss

```diff
@@ -2,26 +2,26 @@
 // ====================
 
 .notification {
-  padding: 30px 30px 0 30px;
+  padding: ($baseline*1.5) ($baseline*1.5) 0 ($baseline*1.5);
 }
 
 .error_msg {
-  margin: 20px;
-  padding: 5px;
+  margin: $baseline;
+  padding: ($baseline/4);
   color: $red;
   border: 1px solid $red;
 
 }
 .cart-errors{
-  background: #FFEEF5;color:#B72667;text-align: center;padding: 10px 0px;
+  background: #FFEEF5;color:#B72667;text-align: center;padding: ($baseline/2) 0;
```

Comment:
woah this file needs a lot more love I think. I made some improvements while I addressed the line break issues in a follow up commit.  If you don't mind one more look through this partial that would be helpful. (mostly it was indentation and spacing where I could parse it, improved use of $baseline and basic color variables.)